src/db/tables/featureFlags.ts

```diff
@@ -8,7 +8,7 @@ import mozlog from "../../utils/log.js";
 const knex = Knex(knexConfig);
 const log = mozlog("DB.flags");
 
-type FeatureFlagDB = {
+export type FeatureFlagDB = {
```

Comment:
I submitted a PR yesterday to provide type definitions for our database tables in the way that's recommended by its documentation, and that allows them to be recognised by the query builder. You might want to align these with that: https://github.com/mozilla/blurts-server/pull/3167

(And also here you might want to change `[string]` to `string[]`.)

Reply:
@Vinnl Good call. I'll modify this after your PR lands

Reply:
Landed! :)